pkg/process/config/yaml_config_test.go

```diff
@@ -0,0 +1,106 @@
+package config
+
+import (
+	coreConfig "github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/stretchr/testify/suite"
+	"net/url"
+	"testing"
+)
+
+type YamlConfigTestSuite struct {
```

Comment:
The tests under "config" seems to test more setting "envs" and general the `config` file. Whereas here the test are more under the `yaml_config` file. I could put them together, but I like the easy usage of a suite 🤔